PWGHF/TableProducer/candidateCreatorDstar.cxx

```diff
@@ -121,6 +122,7 @@ struct HfCandidateCreatorDstar {
     df.setMinRelChi2Change(minRelChi2Change);
     df.setUseAbsDCA(useAbsDCA);
     df.setWeightedFinalPCA(useWeightedFinalPCA);
+    df.setMatCorrType(noMatCorr);
```

Comment:
```suggestion
    df.setMatCorrType(o2::base::Propagator::MatCorrType::USEMatCorrNONE);
```

Reply:
I think this will also go to the new PR.
Currently we have:  
o2::base::Propagator::MatCorrType noMatCorr = o2::base::Propagator::MatCorrType::USEMatCorrNONE;
and within init method:
df.setMatCorrType(noMatCorr);